engine/src/editor/pluginmanager.cpp

```diff
@@ -147,7 +147,13 @@ void PluginManager::init(Engine *engine) {
 
     syncWhiteList();
 
-    loadPlugin(QCoreApplication::applicationDirPath() + "/uikit-editor" + gShared);
+    const char* m_plugin = "";
+#if !defined(__WIN32__) && !defined(__APPLE__)
+    m_plugin = "/../lib/libuikit-editor";
```

Comment:
Ok where is other plugins located?

Reply:
Stand corrected. The plugins folder does exist. Either way, this would need tweaking.

Reply:
bin/platforms - is a qt plugins it's not directly related to Thunder Engine Plugins (modules)

Reply:
Pushing a fix to it now.

Reply:
> bin/platforms - is a qt plugins it's not directly related to Thunder Engine Plugins (modules) but I thought it must be in other place. =/

I call issue tracker! Making a quick new report with my own findings on top of that.

Reply:
Please check this build.
https://github.com/thunder-engine/thunder/actions/runs/9765032247/artifacts/1660653063
It contains expected dirs and files structure

Reply:
Have a work meeting but will look at files while I’m on the bus there. We got this. 😎

Reply:
This is going to be a part of a bigger PR. The uikit-editor code this fixed hid the actual error: Openal linkage error and certain plugins not building successfully.